### PR TITLE
NUX-1286 - Refactor dark props for text input

### PR DESCRIPTION
**Pull request by coleerikson**
#### Screens
text-input-dark

#### Breaking Changes
None I'm aware of.

#### Runway Ticket URL
https://nitro.powerhrg.com/runway/backlog_items/NUX-1286

#### How to test this
Playbook Docs or Nitro Docs

#### Checklist:

- [x] **DEPLOY** Please add the `Milano` label when you are ready for a review
- [x] **SCREENSHOT** Please add a screen shot or two
- [x] **SPECS** Please cover your changes with specs
- [x] **CHANGELOG** Please add an entry on `[Unreleased]` section to every functionality `ADDED`/`CHANGED`/`DEPRECATED`/`REMOVED`/`FIXED`

### CODE REVIEW

**Comment by coleerikson** [comment]
Good catch @kellyeryan. I'm seeing `required` & `disabled` in Rails but not react. Am I missing anything?

**Comment by kellyeryan** [comment]
Maybe validation?

File: app/pb_kits/playbook/pb_text_input/_text_input.jsx [hunk 1]
```diff
@@ -57,26 +60,26 @@ const TextInput = ({
...
       />
-      <div className={classnames('text_input_wrapper', globalProps(props))}>
+      <div className="text_input_wrapper">
         <If condition={children}>
           {children}
           <Else />
           <input
               {...props}
               className="text_input"
+              disabled={disabled}
```
Comment (thestephenmarshall) [inline app/pb_kits/playbook/pb_text_input/_text_input.jsx:72]: I think you can simplify this prop and I think we should add a doc example for disabled state here https://playbook.powerapp.cloud/kits/text_input/react

```suggestion
              disabled
```

Comment (coleerikson) [reply app/pb_kits/playbook/pb_text_input/_text_input.jsx:72]: Without the `={disabled}` the HTML actually just interprets it as a disabled field.

**Pull request merged**